.gitpod.yml

```diff
@@ -12,5 +12,5 @@ tasks:
       
     init: |
       echo "Performing test"
-      ./hyperexecute --user=$LT_USERNAME --key=$LT_ACCESS_KEY --config hyperExecute.yaml --download-artifacts --force-clean-artifacts
+      ./hyperexecute --user=$LT_USERNAME --key=$LT_ACCESS_KEY --config yaml/win/hyperexecute-webdriver-win-autosplit.yaml --download-artifacts --force-clean-artifacts
```

Comment:
Let's remove yaml file that is at the root - [hyperExecute_matrix.yaml](https://github.com/LambdaTest/WebdriverIO-HyperExecute-Sample/blob/main/hyperExecute_matrix.yaml)

Reply:
addressed the comment